### PR TITLE
Feature/covid display

### PR DESCRIPTION
**Pull request by fortuneonyeka**
This is a mobile web application to check a list of metrics created with React and Redux.
The web app has several pages. One page with a list of items that could be filtered by some parameters like the country name. On another page for the item details, the detail page of the country case breaks down.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,61 @@
...
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+2. Now that you have already cloned the repo run the following commands to get the project up and running:
+```sh
+cd space-traveler
+npm install
+npm start
+```
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+1. Next, you can start joining missions and booking rockets. All the changes will be automatically updated on your local server on http://localhost:8080/. You can stop your local server at any time hitting `<Ctrl> + C` on your keyboard.
```
Comment (bimbolabuari) [inline README.md]: Let's remove these redundant comments, This is not related to this project 🙂 

Please check for other occurrences 👍

File: README.md [hunk 2]
```diff
@@ -1,70 +1,61 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This project is [MIT](./LICENSE) licensed.
```
Comment (bimbolabuari) [inline README.md]: As part of the technical requirements, 
> The project should be deployed and accessible online (add a link to the online version of your app to your README file).

Kindly deploy your app using Heroku, Netlify, or GitHub Pages. ✊

File: src/tests/app.test.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-expressions */
```
Comment (bimbolabuari) [inline src/tests/app.test.js]: Let's remove all disabled Linters, Linting helps achieve cleaner and professional code ✊ 


Kindly check for other occurrences 🙂

File: README.md (hunk 2 above)
Comment (bimbolabuari) [inline README.md]: Let's include an "Acknowledgements" section in your README file that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 👍

File: src/index.css [hunk 4]
```diff
@@ -1,13 +1,87 @@
+@tailwind base; /* stylelint-disable-line */
+@tailwind components; /* stylelint-disable-line */
+@tailwind utilities; /* stylelint-disable-line */
```
Comment (bimbolabuari) [inline src/index.css:3]: Let's remove all disabled Linters, Linting helps achieve cleaner and professional code ✊

Kindly check for other occurrences 🙂

File: src/tests/app.test.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  render(
+    <Provider store={store}>
+      <Router>
+        <App />
+      </Router>
+    </Provider>,
+  );
+  const headerText = screen.getByText(/COVID-19 CASE STATISTICS/i);
+  expect(headerText).toBeInTheDocument;
+});
```
Comment (bimbolabuari) [inline src/tests/app.test.js]: There is just one test case 

>Screenshot (337)

Let's provide more integration tests using React Testing Library ✊

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hello @fortuneonyeka, 

You have done a good job so far 🚀  

## Highlights 🎉 

- The project is a single-page application (SPA) built with React and Redux. 🎉 
- The data retrieved from the API is stored in the Redux store. 🚀 
- You filter the data that you retrieve from the API using a Filter stateless component. 👍 

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

- [**Mandatory**] You should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:
   - a description of the project.
   - a demo of the project features.
   - you should also highlight some interesting piece of code or something you built that you are very proud of.
*For recording the video you can use tools like [Loom](https://www.loom.com) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.*

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/tests/app.test.js (hunk 5 above)
Comment (fortuneonyeka) [reply src/tests/app.test.js]: You needed to press a to run all the tests

File: src/index.css (hunk 4 above)
Comment (fortuneonyeka) [reply src/index.css:3]: Tailwind isn't recognized by linters. it can only work with linters disabled.
You can check tailwind documation here [tailwind](https://tailwindcss.com/docs/installation)

**Review by rukundoeric — APPROVED** [review]
Hi @fortuneonyeka,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by fortuneonyeka** [comment]
Thank you very much
